.ci/jenkins_job_build.sh

```diff
@@ -123,6 +123,9 @@ mkdir -p "${GOPATH}"
 export GOROOT="/usr/local/go"
 export PATH=${GOPATH}/bin:/usr/local/go/bin:/usr/sbin:/sbin:${PATH}
 
+# Support Golang 1.16.x
+export GO111MODULE="auto"
```

Comment:
@devimc Can you add in some explanation why this change is required. I see that GO111MODULE is set to on with go 1.16. Would be good to reflect the reasoning behind this in the commit message.

Reply:
fixed, thanks